_posts/hi/newsletters/2022-08-10-newsletter.md

```diff
@@ -0,0 +1,150 @@
+---
+title: 'Bitcoin Optech Newsletter #212'
+permalink: /hi/newsletters/2022/08/10/
+name: 2022-08-10-newsletter-hi
+slug: 2022-08-10-newsletter-hi
+type: newsletter
+layout: newsletter
+lang: hi
+---कम
```

Comment:
These characters after the `---` cause issues with the parsing of this page and thus the page didn’t show up as a translation correctly. Im pushing a small change to remove them. @shaavan can you confirm the translation is still correct? (I used google translate and it seems correct with the characters removed, but please let me know)

Reply:
Hi @bitschmidty!

Thanks for finding the cause of the parsing error. I must have missed it after the `make production` worked successfully for the translations.
I have verified that the meaning of the text remains unchanged after removing this word. 
Thanks again, @bitschmidty!